tracer/src/Datadog.Trace/DiagnosticListeners/AspNetCoreDiagnosticObserver.cs

```diff
@@ -421,10 +412,8 @@ private static Span StartMvcCoreSpan(
 
         private void OnHostingHttpRequestInStart(object arg)
         {
-            var tracer = CurrentTracer;
-            var security = CurrentSecurity;
-            var shouldTrace = tracer.CurrentTraceSettings.Settings.IsIntegrationEnabled(IntegrationId);
-            var shouldSecure = security.AppsecEnabled;
+            var shouldTrace = _tracer.CurrentTraceSettings.Settings.IsIntegrationEnabled(IntegrationId);
+            var shouldSecure = _security.AppsecEnabled;
```

Comment:
While you're here, can we rename these "should" variables?
Since you asked, I'd prefer `integrationEnabled` and `appsecEnabled` 😁